tests/integration/golang/migrations/migrations_test.go

```diff
@@ -0,0 +1,209 @@
+package migrations
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"gorm.io/driver/postgres"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+
+	"github.com/G-Research/fasttrackml/pkg/database"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type MigrationsTestSuite struct {
+	suite.Suite
+}
+
+func TestMigrationsTestSuite(t *testing.T) {
+	t.Skip()
+	suite.Run(t, new(MigrationsTestSuite))
+}
+
+func (s *MigrationsTestSuite) TestMigrations() {
+	type Experiment struct {
+		Name           string `gorm:"type:varchar(256);not null;unique"`
+		ExperimentID   int32  `gorm:"column:experiment_id;not null;primaryKey"`
+		LifecycleStage string `gorm:"type:varchar(32);check:lifecycle_stage IN ('active', 'deleted')"`
+	}
```

Comment:
why not use existing model definitions?

Reply:
(could use `Omit` or `Select` to limit the fields, so current models works with older tables -- I think)